src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/53.diff.sql

```diff
@@ -0,0 +1,1223 @@
+        EXECUTE dbo.LogEvent @Process=@SP,@Status='Error';
+        THROW
+    END CATCH
+    SET @logTime = getUTCdate()
+    EXECUTE dbo.LogEvent @Process='DeleteSearchParamData',@Status='End',@Start=@logTime
+    EXECUTE dbo.LogEvent @Process='Verification',@Status='Start',@Start=@logTime
+    DECLARE @origNumberSearchParamRows bigint = 0
+            ,@tmpNumberSearchParamRows bigint = 0
+            ,@origQuantitySearchParamRows bigint = 0
+            ,@tmpQuantitySearchParamRows bigint = 0
+            ,@origTokenNumberNumberCompositeSearchParamRows bigint = 0
+            ,@tmpTokenNumberNumberCompositeSearchParamRows bigint = 0
+            ,@origTokenQuantityCompositeSearchParamRows bigint = 0
+            ,@tmpTokenQuantityCompositeSearchParamRows bigint = 0
+
+        SELECT @origNumberSearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('NumberSearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @origQuantitySearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('QuantitySearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @origTokenNumberNumberCompositeSearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('TokenNumberNumberCompositeSearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @origTokenQuantityCompositeSearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('TokenQuantityCompositeSearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @tmpNumberSearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('Tmp_NumberSearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @tmpQuantitySearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('Tmp_QuantitySearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @tmpTokenNumberNumberCompositeSearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('Tmp_TokenNumberNumberCompositeSearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @tmpTokenQuantityCompositeSearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('Tmp_TokenQuantityCompositeSearchParam')
+            AND index_id IN (0,1)
+
+    DECLARE @message varchar(100) = '';
+    IF (@origNumberSearchParamRows != @tmpNumberSearchParamRows)
+        BEGIN
+            SET @logTime = getUTCdate()
+            set @message = CONCAT(@origNumberSearchParamRows, @tmpNumberSearchParamRows,'NumberSearchParam rows does not match')
+            EXECUTE dbo.LogEvent @Process='Verification',@Status=@message,@Start=@logTime
+            RETURN;
```

Comment:
>RETURN [](http://example.com/codeflow?start=12&length=6)

THROW here instead of return.

Reply:
Fixed. Simple throw is giving me error, so added random error code, message and severity